cli/src/commands/codegen/index.ts

```diff
@@ -17,17 +17,29 @@ export default class Codegen extends BaseCommand {
   static examples = [];
 
   static flags = {
-    ...this.commonFlags
+    ...this.commonFlags,
+    databaseURL: this.databaseURLFlag,
+    branch: this.branchFlag,
+    output: Flags.string({
+      name: 'output',
```

Comment:
Oclif.Flags does not seem to have a property called `name`. Changing to `o` shorthand.

```suggestion
      char: 'o',
```